app/audit/AuditService.scala

```diff
@@ -58,7 +61,11 @@ class AuditService @Inject() (auditConnector: AuditConnector)(implicit ec: Execu
           "sortCode" -> journey.bankDetails.sortCode
         ),
         "installments" -> Json.obj(
```

Comment:
Just wondernig. 
Perhaps we should just serialise the `PaymentSchedule` to json and put that in the audit here instead for this `installments` field?